src/Admin/OnboardingWizardSubmenu.php

```diff
@@ -40,7 +49,7 @@ public static function get_registration_action() {
 	 */
 	public function register() {
 		add_submenu_page(
-			AMP_Options_Manager::OPTION_NAME,
+			'',
```

Comment:
With this change the page title for the wizard is no longer shown:

![image](https://user-images.githubusercontent.com/16200219/87463281-b0661080-c600-11ea-9551-093c4b0064be.png)

I'd suggest reverting it back to `AMP_Options_Manager::OPTION_NAME`:

```suggestion
			AMP_Options_Manager::OPTION_NAME,
```

Reply:
With that suggestion, we could perhaps use CSS to hide the submenu option,

Reply:
Good catch. I think I will use the `admin_title` hook, though. That would feel a little more solid than hiding with cSS.